README.md

```diff
@@ -1,4 +1,69 @@
-### d8 build scripts
+## d8 build scripts
 
 The repository is part of the [Compiler Explorer](https://godbolt.org/) project. It builds
 the docker images used to build the d8 runtime/compiler/tooling used on the site.
+
+### Using the Dockerfile to make a d8 build locally
+
+You need to have docker installed on your machine. If you do not have docker installed you can
+get it from [here](https://docs.docker.com/engine/install/)
+
+After doing this, run the following command in the root directory of this repository:
+
+```
+docker build -t d8:trunk .
+```
+This will create a container in your machine named `d8:trunk`
+
+You can check if the container has been built by running the following command:
+```
+docker images
+```
+You should see the image listed as follows:
+```
+REPOSITORY   TAG        IMAGE ID       CREATED        SIZE
+d8           trunk      fcd7f69f0e42   3 weeks ago    8.31GB
+```
+
+Once you have this, you can run a container with this image using the following command:
+```
+docker run -d -t d8:trunk
```

Comment:
I really wouldn't run it this way myself. There's no need to have a detached session running - this is like running a UNIX shell as a `daemon` and then connectint to it, with all the attendant problems of remembering to stop and remove it after.

Instead if you run:

```sh
docker run --rm -it d8:trunk bash
```

You wlil get dropped into a fresh, temporary container, running `bash` interactively. The container is removed as soon as that bash shell exits. Then there's no need to `docker exec` or `docker ps`

Reply:
This makes sense. I use the docker desktop app and this lets me use the UI to see the running container and open a terminal from there if I choose. However, I know that my choice of windows as my working machine is a weird one and I will update it to this version.